### ISSUE TITLE
ComplexParalellSplitJoinTest is failing

### ISSUE THREAD

**Issue by apodhrad** [jbosstools/jbosstools-integration-stack-tests#599]
Slave: rewok

Error Message

Timeout after: 10 s.: the number of edit parts is 37

Stacktrace

org.jboss.reddeer.common.exception.WaitTimeoutExpiredException: Timeout after: 10 s.: the number of edit parts is 37
	at org.jboss.reddeer.common.wait.AbstractWait.timeoutExceeded(AbstractWait.java:173)
	at org.jboss.reddeer.common.wait.AbstractWait.wait(AbstractWait.java:126)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:91)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:61)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:46)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:34)
	at org.jboss.reddeer.common.wait.WaitUntil.<init>(WaitUntil.java:25)
	at org.jboss.tools.bpmn2.reddeer.GEFProcessEditor.addConnectionFromPalette(GEFProcessEditor.java:118)
	at org.jboss.tools.bpmn2.reddeer.editor.Element$ConnectionAdded.test(Element.java:569)
	at org.jboss.reddeer.common.wait.WaitUntil.stopWaiting(WaitUntil.java:80)
	at org.jboss.reddeer.common.wait.AbstractWait.wait(AbstractWait.java:122)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:91)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:61)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:46)
	at org.jboss.reddeer.common.wait.AbstractWait.<init>(AbstractWait.java:34)
	at org.jboss.reddeer.common.wait.WaitUntil.<init>(WaitUntil.java:25)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.connectTo(Element.java:300)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.connectTo(Element.java:289)
	at org.jboss.tools.bpmn2.ui.bot.complex.test.testcase.ComplexParalellSplitJoinTest.model(ComplexParalellSplitJoinTest.java:54)

**Comment by apodhrad** [comment]
![org jboss tools bpmn2 ui bot complex test testcase complexparalellsplitjointest jbpm6complextest](https://cloud.githubusercontent.com/assets/2270674/12711378/a7378fea-c8bf-11e5-9520-215c7804fb6e.png)

**Comment by apodhrad** [comment]
@jomarko could you have a look at it, please?

**Comment by tsedmik** [comment]
Another error with ComplexParalellSplitJoinTest:
```
java.lang.NullPointerException: PaletteViewer cannot be null!
	at org.jboss.tools.bpmn2.reddeer.DefaultPalette.<init>(DefaultPalette.java:32)
	at org.jboss.tools.bpmn2.reddeer.ProcessEditorView.<init>(ProcessEditorView.java:58)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.setUp(Element.java:127)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.<init>(Element.java:97)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.<init>(Element.java:79)
	at org.jboss.tools.bpmn2.reddeer.editor.Element.<init>(Element.java:70)
	at org.jboss.tools.bpmn2.reddeer.editor.jbpm.ElementWithParamMapping.<init>(ElementWithParamMapping.java:20)
	at org.jboss.tools.bpmn2.reddeer.editor.jbpm.activities.Task.<init>(Task.java:26)
	at org.jboss.tools.bpmn2.reddeer.editor.jbpm.activities.UserTask.<init>(UserTask.java:31)
	at org.jboss.tools.bpmn2.ui.bot.complex.test.testcase.ComplexParalellSplitJoinTest.model(ComplexParalellSplitJoinTest.java:40)
```

=> SmokeTests are failing due to this error

**Comment by jomarko** [comment]
@apodhrad or @tsedmik, you can close this issue, it will be resolved in thanks of [1]

[1]
https://github.com/jbosstools/jbosstools-integration-stack-tests/issues/643

**Comment by apodhrad** [comment]
closed, it will be resolved at #643
